platform/security/idp/security-idp-server/src/main/java/org/codice/ddf/security/idp/server/LogoutState.java

```diff
@@ -33,8 +35,13 @@
 
   private boolean partialLogout = false;
 
-  public LogoutState(Set<String> spDescriptors) {
+  private List<String> sessionIndexes = Collections.emptyList();
+
+  public LogoutState(Set<String> spDescriptors, List<String> sessionIndexes) {
     this.spDescriptors = Collections.synchronizedSet(spDescriptors);
```

Comment:
❓ Do we really need this collection to be synchronized? The two methods that access this collection already are.

Reply:
If Ryan says it can go, it can go. Removing it.